kyuubi-server/src/main/scala/org/apache/kyuubi/engine/spark/SparkBatchProcessBuilder.scala

```diff
@@ -64,10 +66,40 @@ class SparkBatchProcessBuilder(
     buffer.toArray
   }
 
+  private def sparkFilesConf(sparkConf: Map[String, String]): Map[String, String] = {
```

Comment:
why not just using spark.files in the very begining?

Reply:
do you mean that remove the `files`, `jars` filed in BatchRequest and using `spark.files` and `spark.jars` configuration item instead?

Reply:
yes, 
-  these 2 APIs do the same thing, 
- users don't how we deal with them
- spark behavior may change as it is undocumented and underlying and hardcore.(for the part how we merge these)

Reply:
got it, thanks